src/environments/environment.ts

```diff
@@ -5,7 +5,10 @@
 export const environment = {
   production: false,
   APIUrl:"https://localhost:5001/api",
-  APIFileUrl:"https://localhost:6001/api/fileservice"
+  APIFileUrl:"https://localhost:6001/api/fileservice",
+  APIPhotoUrl:"https://localhost:7001/api/photoservice",
+  APIVideoUrl:"https://localhost:8001/api/videoservice"
+  
```

Comment:
General comments)
1. No "any" type 
2. all methods should have the return type
3. use private methods in places where it needed 
4. unsubscribe, no memory leaks
5. all components should have OnPush update strategy
6. for CSS use some naming convention
7. install prettier, and fix all lint issues) P.S. prettier --write command
8. Keep your methods small and easy)
9. Use KISS Principe))